theme/official_themes.go

```diff
@@ -0,0 +1,33 @@
+package theme
+
+import (
+	"os"
+	"path"
+
+	git "gopkg.in/src-d/go-git.v4"
+)
+
+const officialThemesRepo = "https://github.com/ArmanMazdaee/test-resumic-themes.git"
+
+func getOfficialThemesDir(cacheDir string) string {
+	return path.Join(cacheDir, "official_themes")
+}
+
+func pullOfficialThemes(root string) error {
+	if _, err := os.Stat(root); os.IsNotExist(err) {
+		_, err := git.PlainClone(root, false, &git.CloneOptions{
```

Comment:
Also currently this would pull in all themes. It might be better, that we pull in the index and then on demand only pull the specific submodule, that corresponds to the theme the user wants to use. Happy to hear your thoughts. Also we can do this in a follow on PR, if you think that is a better approach.